submissions/pozzitive11/js-world-oop/index.js

```diff
@@ -0,0 +1,65 @@
+class Inhabitant {
+  constructor(species, name, gender, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+  }
+
+  getProperties() {
+    return `${this.species}; My name ${this.name}; ${this.gender}; ${this.saying}`;
+  }
+}
+class Animal extends Inhabitant {
+  constructor(name, gender, saying) {
+    super(name, gender, saying);
```

Comment:
`Inhabitant` class constructor expects `species` as the first argument. Call of `super` doesn't send any.